### ISSUE TITLE
Move the VocBench and ShowVoc instances on the international OVH account

### ISSUE THREAD

**Issue by RachL** [datafoodconsortium/standard#35]
...and find a way to share the access among international members

**Comment by lecoqlibre** [comment]
I would say we need half a day (3.5 - 4 hours) to migrate the two instances on the international OVH account and share access with the international members.

We might need to create a new server and assign it to the international account unless we can just detach and attach an existing server to another account.

**Comment by imbrou** [comment]
Hi @RaggedStaff! I am also migrating to the following domain names:

`vocbench.dfc-standard.org` (+ trying to strip `/vocbench3/`)
`showvoc.dfc-standard.org` (+ trying to strip `/showvoc/`)

If for any reason we might want to keep one or both at `datafoodconsortium.org`, here is room for dscussion!

**Comment by RaggedStaff** [comment]
> If for any reason we might want to keep one or both at `datafoodconsortium.org`, here is room for discussion.

Nope. That sounds great to me! We'll need to update references in the Gitbook to match. Unless you can put a redirect in place for now? 🙏🏻

**Comment by imbrou** [comment]
Hey! I can't migrate directly the binary data from the old instances to the new. It looks like there are internal incompatibilities between the old data schema and the new versions data schema, and SV and VB throw errors.

All data is still safe on datafoodconsortium.org (OVH fr).

On the new instances at dfc-standard.org (OVH international), I have started populating new empty VB/SV instances, importing SKOS & OWL data from the repos.

To my VB beginner understanding, the only thing lost in this process are the user accounts.

Can you please confirm that there is no other critical data in the internal (SemanticData) storage of SV/VB?

For instance, is VB totally relying on the git repos for versioning, or does it have some internal versioning mechanism that I am not aware of? With an internal storage of draft versions, for example?

**Comment by RaggedStaff** [comment]
There isn't versioning exactly,  but @Alcoz did enable some sort of change control functionality,  where changes have to be approved (by a manager I think). I think we want to keep that,  for when we have wider community contributions...  and that isn't data exactly,  more VB config.

**Comment by imbrou** [comment]
🎉 ShowVoc and VocBench have just been migrated to:

https://showvoc.dfc-standard.org/
https://vocbench.dfc-standard.org/

I had to recreate all user accounts, so **all ShowVoc & VocBench users must get a new password**. I have sent passwords to the admin users in a Slack DM. I will communicate new passwords to other users soon. Feel free to ping me if… you want it sooner than soon! 🙂

**Comment by lecoqlibre** [comment]
Great work, thanks @imbrou.

More details in https://github.com/datafoodconsortium/infra/tree/main/vocbench-showvoc.
